tools/webpacks/grid.webpack.mjs

```diff
@@ -91,7 +91,7 @@ const webpacks = FSLib
                 }),
                 new ProductMetaPlugin({
                     productName: 'Grid',
-                    productVersion: '1.0.0' // We need to discuss it (DD)
+                    productVersion: '1.0.0' // Update version before releasing
```

Comment:
Wouldn't it be more convenient to store every version and for each product in `build-properties.json`?

Reply:
`build-properties.json` refers directly to core products, I don't want to make changes for it that would change its purpose. One of those things that would be worth rebuilding in the long run to tidy up the monorepo to support several different products with different release procedures and different versions, but here a mixed team consisting of members from each current one would be useful.

Reply:
I kinda have a mixed feeling about that because we provide the `release` flag with a version and we also have to change it here. 
Looks like a possible point where some ambiguity might occur.
If there is no better way to handle that, we can leave it :)